README.md

```diff
@@ -89,6 +90,16 @@ The compilation can be speeded up by specifying the `PYTORCH_ROCM_ARCH` variable
 
 For ROCm systems, the packages `rocsparse-dev`, `hipsparse-dev`, `rocthrust-dev`, `rocblas-dev` and `hipblas-dev` are required to build.
 
+#### On Intel Gaudi systems
+
+To install from source for Intel Gaudi HPUs, set environment variable `BUILD_CUDA_EXT=0` to disable CUDA pytorch extension building. Example:
```

Comment:
I think we should say Gaudi 2, can you ask Sherin?

Reply:
If it's supported on Gaudi 2 only, then we should specify it.
If this is the first mention of Gaudi, then we should write Intel® Gaudi® 2 AI accelerator

Change to: set the `BUILD_CUDA_EXT=0` environment variable to disable  building the CUDA PyTorch extension. Example: